core/ui/DefaultSearchResultList.tid

```diff
@@ -9,7 +9,7 @@ caption: {{$:/language/Search/DefaultResults/Caption}}
 
 //<small>{{$:/language/Search/Matches/All}}</small>//
 
-<$list filter="[!is[system]search{$(searchTiddler)$}sort[title]limit[250]]" template="$:/core/ui/ListItemTemplate"/>
+<$list filter="[!is[system]search{$(searchTiddler)$}sort[title]limit[250]] -[!is[system]search:title{$(searchTiddler)$}]" template="$:/core/ui/ListItemTemplate"/>
```

Comment:
I think, the `limit[250]` should be at the end, after the list is shortened.

Reply:
if that's possible.

Reply:
you're right I think, it's fixed